paasta_tools/frameworks/task_store.py

```diff
@@ -40,12 +40,12 @@ def __init__(
         offer=None,
```

Comment:
A note here that `mypy` would actually complain about this `__init__` if it were being typechecked -- the annotations for `mesos_task_state`, `is_draining`, and `is_healthy` are non-nullable, but these parameters are `Optional`.  fortunately (?) mypy doesn't typecheck functions without annotations so this isn't _currently_ failing (but it might be a bug / surprise in the future!)